README.md

```diff
@@ -39,8 +40,9 @@ To ensure you have the correct dependencies run `yarn install` in the command li
 To run code linting enter `yarn lint` in the command line.
 
 ## Testing:
-To run e2e tests enter `yarn test` in the command line.
+To run e2e tests enter `codeceptjs run` in the command line
 
-## License
+## Service:
+See [fpl-serivice](#service/README.md) for more information.
```

Comment:
Please remove `#` as per https://help.github.com/articles/basic-writing-and-formatting-syntax/#relative-links